website/docs/reference/node-selection/methods.md

```diff
@@ -126,3 +126,15 @@ $ dbt run --select +exposure:weekly_kpis                # run all models that fe
 $ dbt test --select +exposure:*                         # test all resources upstream of all exposures
 $ dbt ls --select +exposure:* --resource-type source    # list all sources upstream of all exposures
 ```
+
+### The "result" method
+<Changelog>New in v1.0.0</Changelog>
+
+The `result` method is related to the `state` method described above, and can be used to select resources based on their result state from a prior run. Note that one of the dbt commands [`run`, `test`, `build`, `seed`] must have been performed in order to create the result on which a result selector operates. You can use `result` selectors in conjunction with the `+` operator.
+
+```bash
+$ dbt run --select result:error # run all models that generated errors on the prior invocation of dbt run
+$ dbt test --select result:fail # run all tests that failed on the prior invocation of dbt test
+$ dbt build --select result:fail # run all the models associated with failed tests from the prior invocation of dbt build
```

Comment:
I think this might need to be:
```suggestion
$ dbt build --select 1+result:fail # run all the models associated with failed tests from the prior invocation of dbt build
```